src/_scss/components/checkbox/1_accordionCheckbox.scss

```diff
@@ -0,0 +1,50 @@
+// TODO: rename file once 11456 is merged into QAT and into this branch
```

Comment:
was the file renamed? can this comment be deleted?

Reply:
It has been and it can be. I'll delete it!